### PR TITLE
Dashboard landing: Add a dashboard plus ff section

### PR DESCRIPTION
**Pull request by Yndira-E**
## Description

In an effort to highlight that the Dashboard is even better when combined with the FlowFuse platform, I've added a new section to the landing page.

## Related Issue(s)

[#2279](https://github.com/FlowFuse/website/issues/2279)

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by Yndira-E** [comment]
Hi @Hasmin-AC, wdyt?

I know we have "Flexible deployment with FlowFuse" as part of the previous section, but I wanted to add another one dedicated exclusively to the benefits of using FF + Dashboard and being a bit more specific.

File: src/product/dashboard.njk [hunk 1]
```diff
@@ -64,9 +65,23 @@ meta:
             </li>
             {% endfor %}
         </ul>
+        <!-- FlowFuse + Dashboard -->
+        <div class="ff-blue-card">
+            <h2 class="text-center w-full md:text-left mb-10"><span class="text-indigo-600">Unlock the Full Potential</span> with FlowFuse</h2>
```
Comment (Hasmin-AC) [inline src/product/dashboard.njk:70]:
```suggestion
            <h2 class="text-center w-full md:text-left mb-10"><span class="text-indigo-600">Unlock Your Full Potential</span> with FlowFuse</h2>
```

**Review by Hasmin-AC — COMMENTED** [review]
Looks good @Yndira-E . Ideally each of the bolded text takes you to an article/video about it. 

@sumitshinde-84 can you suggest or create a blog if it doesn't exist to link to for these key points?

**Comment by sumitshinde-84** [comment]
here are some resources:

- build personalized Dashboards for each of your users: https://www.youtube.com/watch?v=rl6uwV2PhOc | https://flowfuse.com/blog/2024/01/dashboard-2-multi-user/ | https://flowfuse.com/blog/2024/04/how-to-build-an-application-with-node-red-dashboard-2/
- Easy integration with FlowFuse Devices:- https://www.youtube.com/watch?v=pCYzJGVkG8s
- trustworthy security out of the box: - https://www.youtube.com/watch?v=TY9dhT_tAT0

**Comment by sumitshinde-84** [comment]
here are some resources 

- build personalized Dashboards for each of your users: https://www.youtube.com/watch?v=rl6uwV2PhOc | https://flowfuse.com/blog/2024/01/dashboard-2-multi-user/ | https://flowfuse.com/blog/2024/04/how-to-build-an-application-with-node-red-dashboard-2/
- Easy integration with FlowFuse Devices:- https://www.youtube.com/watch?v=TY9dhT_tAT0
- trustworthy security out of the box: - https://www.youtube.com/watch?v=pCYzJGVkG8s

**Comment by Yndira-E** [comment]
Links added!!

**Pull request merged**